packages/terra-alert/src/Alert.jsx

```diff
@@ -194,6 +203,15 @@ const Alert = ({
     </div>
   );
 
+  useEffect(() => {
+    const isAlert = role === 'alert' || defaultRole === 'alert';
+    if (!disableAlertActionFocus && isAlert && action && alertRef?.current?.focus) {
```

Comment:
nit: Since the disableAlertActionFocus check is more likely to pass, should we put isAlert first here to break out earlier?

Reply:
Good call, I also moved action since we only focus when there's an action element and added comment here https://github.com/cerner/terra-core/pull/3797/commits/217745d6cf1a459d940744060cc3dd15c1828016